test/server/BUILD

```diff
@@ -307,6 +307,9 @@ envoy_cc_test(
         "//source/common/network:socket_option_lib",
         "//source/common/network:utility_lib",
         "//source/common/protobuf",
+        "//source/common/secret:secret_manager_impl_lib",
+        "//source/extensions/filters/http/oauth2:config",
```

Comment:
We generally try to avoid bringing extensions into core tests, since it prevents the import into Envoy versions without those extensions. Can we move this test into the extension specific test suite?

Reply:
removed these changes